README.md

```diff
@@ -54,7 +54,7 @@ A Travis-CI file has been included to enable CI builds. The correct configuratio
   * What do you want to use as your public directory? -> `dist`
   * Configure as a single-page app (rewrite all urls to /index.html)? -> `Yes`
   * What Firebase project do you want to associate as default?  -> **your Firebase project name**
-5. Build Project: `npm run build`
+5. Build Project: `npm run build:prod`
```

Comment:
Yup this should be changed. The plan in future versions is to just have it be the `build` command